test/parallel/test-cluster-inspector-debug-port.js

```diff
@@ -27,6 +27,8 @@ if (cluster.isMaster) {
   fork(4, ['--inspect', '--debug']);
   fork(5, [`--debug=${debuggerPort}`, '--inspect']);
   fork(6, ['--inspect', `--debug-port=${debuggerPort}`]);
+  fork(7, [`--inspect-port=${debuggerPort}`]);
+  fork(8, ['--inspect', `--inspect-port=${debuggerPort}`]);
```

Comment:
@bnoordhuis @Fishrock123 is this disposition acceptable to you, or do you think that #11420 needs to be addressed before this lands?